Documentation/docs-mobile/building-apps/build-items.md

```diff
@@ -129,6 +129,35 @@ multiple files, and they will be evaluated in no particular order (so don't
 specify the same environment variable or system property in multiple
 files).
 
+## AndroidGradleProject
+
+`<AndroidGradleProject>` can be used to build and consume the outputs
+of Android Gradle projects created in Android Studio or elsewehere.
+
+The `Include` metadata should point to the root `build.gradle` file that will
+be used to build the project, inside of the top directory of your Gradle project
+which also contains the `gradlew` wrapper scripts.
+
+```xml
+<ItemGroup>
+  <AndroidGradleProject Include="path/to/project/build.gradle.kts" ModuleName="mylibrary" />
+</ItemGroup>
+```
+
+The following MSBuild metadata are supported:
+
+- `%(Configuration)`: The name of the configuration to use to build or assemble
+  the project or project module specified. The default value is `Release`.
+- `%(ModuleName)`: The name of the [module or subproject](https://docs.gradle.org/current/userguide/intro_multi_project_builds.html) that should be built.
+  The default value is empty.
+- `%(OutputPath)`: Can be set to override the build output path of the Gradle project.
```

Comment:
Should this exist publicly?  In what circumstances would/should the developer set this metadata?

Reply:
I didn't see a reason to restrict this, maybe someone would want Gradle project output to go to a less "temporary" location outside of the intermediate output directory.

Reply:
That's fair.